standard/expressions.md

```diff
@@ -1039,27 +1051,27 @@ Given two types `T₁` and `T₂`, `T₁` is a ***better conversion target*** th
 > *Example*: The following examples show overloads that are valid and invalid according to this rule:
 >
 > ```csharp
-> interface I1<T> {...}
-> interface I2<T> {...}
+> interface I1<T> { ... }
+> interface I2<T> { ... }
 >
-> class G1<U>
+> abstract class G1<U>
 > {
->     int F1(U u);               // Overload resulotion for G<int>.F1
->     int F1(int i);             // will pick non-generic
->     void F2(I1<U> a);          // Valid overload
->     void F2(I2<U> a);
+>     abstract public int F1(U u);               // Overload resulotion for G<int>.F1
+>     abstract public int F1(int i);             // will pick non-generic
+>     abstract public void F2(I1<U> a);          // Valid overload
```

Comment:
I'd be tempted to add a blank line between each set of names (so both F1s together, then both F2s, and likewise for F3, F4, F5, F6 in G2.)